docker-compose.yml

```diff
@@ -18,3 +18,15 @@ services:
       - .:/editor:delegated
     ports:
       - "6006:6006"
+    environment:
+      - STORYBOOK_SIGNAL_URL=ws://signal:6000
+  signal:
+    build: .
```

Comment:
reusing the same docker image, because why not? Just change the entrypoint and we'll be fine. Splitting these images can happen later if we get far enough that that is important.